java.lang.UnsatisfiedLinkError: Required library was not found

Raspberry Pi 4 model Bで動作させると、以下のエラーが発生します。

```
vcspeakerkt-app-1  | Caused by: java.lang.RuntimeException: java.lang.UnsatisfiedLinkError: Required library was not found
vcspeakerkt-app-1  |    at com.sedmelluq.lava.common.natives.NativeLibraryLoader.loadWithFailureCheck(NativeLibraryLoader.java:88)
vcspeakerkt-app-1  |    at com.sedmelluq.lava.common.natives.NativeLibraryLoader.load(NativeLibraryLoader.java:69)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.natives.ConnectorNativeLibLoader.loadConnectorLibrary(ConnectorNativeLibLoader.java:21)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.natives.opus.OpusEncoderLibrary.getInstance(OpusEncoderLibrary.java:15)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.natives.opus.OpusEncoder.<init>(OpusEncoder.java:20)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.format.transcoder.OpusChunkEncoder.<init>(OpusChunkEncoder.java:23)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.format.OpusAudioDataFormat.createEncoder(OpusAudioDataFormat.java:59)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.filter.AudioPipelineFactory.createPostProcessors(AudioPipelineFactory.java:58)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.filter.AudioPipelineFactory.create(AudioPipelineFactory.java:35)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.container.wav.WavTrackProvider.<init>(WavTrackProvider.java:39)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.container.wav.WavFileLoader.loadTrack(WavFileLoader.java:85)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.container.wav.WavAudioTrack.process(WavAudioTrack.java:30)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.source.local.LocalAudioTrack.process(LocalAudioTrack.java:45)
vcspeakerkt-app-1  |    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104)
vcspeakerkt-app-1  |    ... 4 common frames omitted
vcspeakerkt-app-1  | Caused by: java.lang.UnsatisfiedLinkError: Required library was not found
vcspeakerkt-app-1  |    at com.sedmelluq.lava.common.natives.NativeLibraryLoader.extractLibraryFromResources(NativeLibraryLoader.java:125)
vcspeakerkt-app-1  |    at com.sedmelluq.lava.common.natives.NativeLibraryLoader.loadInternal(NativeLibraryLoader.java:110)
vcspeakerkt-app-1  |    at com.sedmelluq.lava.common.natives.NativeLibraryLoader.loadWithFailureCheck(NativeLibraryLoader.java:84)
vcspeakerkt-app-1  |    ... 18 common frames omitted
```

arm64など複数アーキテクチャで実行できるように対応が必要です。

- see: https://github.com/jaoafa/JDA-VCSpeaker/pull/228

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -25,6 +25,7 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1")
     implementation("ch.qos.logback:logback-classic:1.3.7")
     implementation("com.sedmelluq:lavaplayer:1.3.77")
+    implementation("com.github.aikaterna:lavaplayer-natives:original-SNAPSHOT")
     implementation("com.uchuhimo:konf:1.1.2")
     implementation("com.github.ajalt.clikt:clikt:4.0.0")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")

